deployment/docker/auth-vm-cluster.yml

```diff
@@ -2,18 +2,16 @@
 # see https://docs.victoriametrics.com/victoriametrics/vmauth/#load-balancing
 # Note: if username and password are changes, please update the Grafana datasource configuration
 # and check other places where these credentials are used.
-users:
-  - username: "foo"
```

Comment:
I don't see the point of having authorized user in the example, let me know if you think so, then both vmagent and grafana auth config need to be applied.